### DISCUSSION TITLE
Official mobile apps

### DISCUSSION THREAD

**Discussion by jljl1337** [hcengineering/platform#7183] [Ideas]
Is there any timeline for mobile app development?

**Comment by otozyon** [comment]
This is very much needed. Being able to manage projects on mobile is very important. I don't always carry my notebook around. A dedicated app would be great. But better support for mobile on web would already be a good start. At least make sure that the UI doesn't break on smartphone screens.

**Comment by ycsh-w** [comment]
+1 to this we'd love to hear more about the updates on the mobile app development.

**Comment by epireve** [comment]
This would be top on my wishlist!

**Comment by rao-manoj** [comment]
+1

I am seriously considering switching from linear to huly for all my client projects. No mobile app is the only deal breaker

**Comment by Anaxagoras-bc** [comment]
I've been looking at team chat apps with project management for the past week, and Huly is the best one i've found so far, it checks most of my boxes, but not having a mobile app is a dealbreaker.  Even if the mobile app was primarily chat, it would be good enough.

A desktop mac/windows/linux app would also be preferred to a web brwoser, even if it just winds up being an electron app.

**Comment by Rnbsov** [comment]
+1

**Comment by UncleDoomVSSP** [comment]
+1 on this
